src/routes/UserSettings/UserSettings.js

```diff
@@ -12,20 +11,7 @@ import FullPageModal from 'routes/FullPageModal'
 import { PROJECT_CONTRIBUTIONS } from 'config/featureFlags'
 import './UserSettings.scss'
 
-const { object, func } = PropTypes
-
 export default class UserSettings extends Component {
-  static propTypes = {
-    currentUser: object,
-    onClose: func
-  }
-
-  componentDidMount () {
-    const { currentUser } = this.props
-    this.props.fetchForCurrentUser()
```

Comment:
we dont need this even if we just go straight to the user settings URL?

Reply:
Correct, and tested. It is reliably handled by the auth logic on the way in. Ah, actually, I forgot. It likely can/should be removed here in CommunitySettings as well https://github.com/Hylozoic/hylo-evo/blob/50a634204b8e3b1e4ca9713306edfa7c63e4f385/src/routes/UserSettings/CommunitySettingsTab/CommunitySettingsTab.js#L38. I'll do that shortly and include it in this branch.

Reply:
i guess the only reason to have these is if there is some change that happens that has not propagated to the front-end yet for some reasons so then looking at these tabs is showing out of date information. im not sure if that is a really valid concern. But for example if someone has two tabs open and makes a change in one and then goes to the settings page in the other.

Reply:
thoughts? even if we did keep them they should not block loading of the UI

Reply:
That is why they would have existed yes, and I've finger tested the cases I could come-up with which could potentially generate that condition and didn't ever find a problem. Regardless, I think its' fair to say that this SHOULDN'T ever happen if we have faith in our Redux ORM use and if we employ it consistent throughout the app. If it does happen it'd be better to catch it as a bug in this area than in more subtle places, and again I don't think there is are issues in this case. 

Finally, to load  this here "just in case" while not block editing has the result of updating  the component while it's being edited overwriting any not yet saved edits. We could of course build around that in some way but that would add mess for what I think in this case is no benefit.

Reply:
Also keep in mind that the things are edited on the Profile screen aren't available to change anywhere within the app outside of this screen. This is true with affiliations as well, and for memberships I think it's reliable as it's something we more fundamentally can't afford to have out of sync elsewhere in the app... Make sense?

Reply:
To be fair there is at least one edge case: if you have multiple Hylo windows open and/or mobile and you're making edits in this are on both at the same time without reloading. I think that is a tolerable vulnerability with low consequences.

Reply:
yeah I hear all that. im good with this